lua/starfall/libs_sv/prop_sent.lua

```diff
@@ -0,0 +1,1595 @@
+-- Simple helper function that provides model if it doesnt exist
+local function registerSent(class, data)
+	data[1].model = data[1].model or {"Model", TYPE_STRING, "models/props_junk/watermelon01.mdl"}
```

Comment:
`if not data[1].model then data[1].model = {"Model", TYPE_STRING, "models/props_junk/watermelon01.mdl"} end`